data/src/main/java/me/androidbox/data/alarm_manager/AlarmReceiver.kt

```diff
@@ -25,15 +29,26 @@ class AlarmReceiver : BroadcastReceiver() {
             }
         }
     }
-    
+
     private fun createAndShowNotification(context: Context, agendaId: String, title: String, description: String) {
         val notificationManager = getNotificationManager(context)
 
+        /** TODO Only Events for the moment, tasks and reminders coming soon... */
+        val agendaIntent = Intent(
+            Intent.ACTION_VIEW,
+            "$EVENT_DEEPLINK".replace("{id}", agendaId).toUri())
```

Comment:
I would replace agendaId with something like agendaItemId

Since you are refering to the id of an item in the agenda